### PR TITLE
Remove duplicate word in bip-0085.mediawiki

### PR DESCRIPTION
**Pull request by nau**
Fix doublicated word

### CODE REVIEW

**Comment by luke-jr** [comment]
@ethankosakovsky

**Review by Skyeenicole — APPROVED** [review]

**Review by illesdavid — APPROVED** [review]

**Review by Skyeenicole — APPROVED** [review]
How does withdrawal work?

**Review by Dushman777 — APPROVED** [review]

**Review by Skyeenicole — APPROVED** [review]

**Review by Dushman777 — COMMENTED** [review]
.......

**Review by Dushman777 — COMMENTED** [review]
,🧐🧐🧐

**Review by Skyeenicole — APPROVED** [review]
😵‍💫

File: bip-0085.mediawiki [hunk 1]
```diff
@@ -364,7 +364,7 @@ The resulting RSA key can be used to create a GPG key where the creation date MU
 
 Note on GPG key capabilities on smartcard/hardware devices:
 
-GPG capable smart-cards SHOULD be be loaded as follows: The encryption slot SHOULD be loaded with the ENCRYPTION capable key; the authentication slot SHOULD be loaded with the AUTHENTICATION capable key. The signature capable slot SHOULD be loaded with the SIGNATURE capable key.
```
Comment (Skyeenicole) [inline bip-0085.mediawiki:367]: ]()

**Pull request merged**